Seohee/Lv2/카펫.js

```diff
@@ -0,0 +1,27 @@
+function getCarpets(size) {
+    let rows = [];
+    // 가로 길이들 담기
+    for (let i = 1; i <= size; i += 1) {
+        if (size % i === 0) rows.push(i);
+    }
+    rows.reverse();
```

Comment:
```suggestion
    for (let i = size; i > 0; i -= 1) {
        if (size % i === 0) rows.push(i);
    }
```
for문을 이용해 배열을 만들고 reverse를 해준다면 for문의 순서를 바꿔주는게 성능면에서 좋아보여요.